packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -292,25 +296,30 @@ export const TrackScreenDetailsTile = ({
     [track]
   )
 
+  const currentQueueItem = useSelector(getCurrentQueueItem)
```

Comment:
This logic is scary but it is copied verbatim from the web flow now, which didn't have the bug
https://github.com/AudiusProject/audius-protocol/blob/dc8c41c8b3e38e15f04dfd68b2f7cdc86e68419d/packages/web/src/pages/track-page/TrackPageProvider.tsx#L245